contents/blog/how-to-fix-your-growth-team.md

```diff
@@ -0,0 +1,155 @@
+
+Because you'll have a clearer idea of what to expect from your experiments, you'll prioritize the ones that are more likely to win.
+
+This approach was made famous by Facebook Messenger's VP of growth Darius Contractor and his [EVELYN framework](https://www.drift.com/blog/secret-to-faster-growth/) (He even shares a [handy sheet with real examples you can copy](https://airtable.com/appzs6HT2ggrJt9pN/shrInOxl0SL1Xmauk/tblTQFEMcXAaEgSpM/viwZessbXpiA8q2jd?blocks=bipMQFWTjqu0HI3wv)).
+
+Alexey says that **one of their biggest wins came from following this approach**:
+
+"At MasterClass, we had one-size-fits-all pricing. We knew we would eventually need to add tiered pricing, but this would be a huge, months-long company effort. We struggled to prioritize it."
+
+Enter EVELYN. Alexey knew they needed a dollar estimate of the impact.
+
+"Instead of building the entire backend, we created a simple [fake door test](/tutorials/fake-door-test) on the frontend. Users could select their preferred tier – but at checkout, we let them know they were being upgraded to the best tier and at the cheapest price. In reality, this was actually the existing one-size-fits-all pricing.
+
+Now we knew how much money the company stood to make! We prioritized it immediately, and it was incredibly easy to ask the rest of the company to work on it for a few months."
+
+## Problem 3: Your experiment win size is low
+
+Your growth team has been around for a few quarters and you've picked all the low-hanging fruit. Your new experiments require more effort AND have less impact. Now what?
+
+[John Egan](https://jwegan.com/growth-hacking/managing-growth-teams-portfolio-step-step-guide/), former head of growth engineering at Pinterest, recommends two levers you can pull:
+
+1. Increase the number of high-quality project ideas.
+2. Make your engineers own their experiments.
+
+### Increase the number of high-quality project ideas
+
+The best way to do this is to create a bottom-up culture where it's the job of **every single person** to come up with ideas. That means every engineer, designer, data scientist, sales rep, marketer, etc.
+
+When it's everyone's job to do this, the number of ideas increases dramatically. Then, by applying the EVELYN framework mentioned above, you can prioritize and distill the list to only the best ideas.
+
+### Make your engineers own their experiments
+
+Every engineer must be responsible for making their experiments as impactful as possible – even if it wasn't originally their idea. 
+
+Empowerment is the key here. When engineers take charge of their experiments, they become more invested in the outcome, leading to better results.
+
+In practice, this means:
+
+- Engineers work to improve the initial hypothesis.
+- Engineers develop further iterations to test. 
+- Removing processes that prevent engineers from making quick decisions on their own.
+- Giving engineers space to fail without being harsh on them.
```

Comment:
It's jarring that you change the format of the article in this section – a format that really works btw.

Also, what constitutes a low win size? This is context readers need to make this useful.